config/initializers/sidekiq.rb

```diff
@@ -1,8 +1,9 @@
 # frozen_string_literal: true
 
 if defined?(Sidekiq) && ENV['SIDEKIQ_REDIS_URL']
+  config.logger.level = Logger.const_get(ENV.fetch('RAILS_LOG_LEVEL', 'info').upcase.to_s)
```

Comment:
`config` is undefined here. I think `logger.level` will need to be set within both the configure_client and configure_server blocks.